static/js/src/advantage/subscribe/checkout/components/Summary/Summary.tsx

```diff
@@ -120,6 +120,12 @@ function Summary({ quantity, product, action, setError }: Props) {
                 : currencyFormatter.format(defaultTotal)}
             </strong>
           </p>
+          <p>
+            <>
+              {total == 0 &&
+                "This is because you have likely already paid for this product for the current billing period."}
+            </>
+          </p>
```

Comment:
There seem to be some other cases total becoming `0`

I see this text too when I make a purchase as a guest
![image](https://github.com/canonical/ubuntu.com/assets/57550290/3e892fab-ecf2-416c-9e24-8ef624cc6745)